docs/content/topics/scripting_advanced.md

```diff
@@ -193,3 +193,34 @@ world
 ```
 
 Job `j1` has our extra command, while `j2` only inherited the defaults from `MyJob`.
+
+
+## Using Docker Within a Brigade Job
+
+It is possible to use Docker inside of a Brigade job. However, you will need to do some extra work. Because a Job must run in privileged mode to use the Docker socket, the method here presents a security risk and should not be allowed for untrusted brigade scripts.
+
+Before you can write scripts that use privileged mode, you will need to set the following permissions on your Brigade project:
+
+```yaml
+allowHostMounts: "true"
```

Comment:
Strictly speaking, `allowHostMounts: "true"`/`dind.docker.enabled =true;` wouldn't necessarily be required for the example below, as the job container runs the docker daemon itself and only really requires `privileged` be set.  A bit of clarification will be added to the review here...

Reply:
Ooo... yeah, it appears that docker.enabled = true actually breaks it